universal-application-tool-0.0.1/app/auth/UatProfile.java

```diff
@@ -64,6 +64,10 @@ public String getClientName() {
     return profileData.getRoles();
   }
 
+  public boolean isUatAdmin() {
+    return profileData.getRoles().contains(Roles.ROLE_UAT_ADMIN.toString());
```

Comment:
question (if-minor): Can this be an exact match instead of a contain check?

Reply:
`getRoles()` returns a set of strings, so I don't think so

Reply:
Ah cool. Thought the return type is a single string. Feel free to ignore this.